.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: node_js
 node_js:
-  - "5"
+  - "6"
+  - "node"
```

Comment:
I'd never seen `node` as a value before. I had to look it up. It is an alias for the "latest stable Node.js release"
 https://docs.travis-ci.com/user/languages/javascript-with-nodejs#Available-Versions